### ISSUE TITLE
RememberBestWeights does not honor the verbose parameter

### ISSUE THREAD

**Issue by jrodrigog** [dnouri/nolearn#301]
Hello.
The _RememberBestWeights_ class does not honor the verbose parameter.
Verbose is hardcoded to 1 in the `__init__` method `self.verbose = 1`.
It should be `self.verbose = verbose`.
Tested with nolearn-0.6.1.dev0.

**Comment by nickmitchko** [comment]
# Fixed with pull #303 
## **Reinstallation:**
### Linux
```BASH
# Python 2
sudo pip install --upgrade --no-deps --force-reinstall nolearn
# Python 3
sudo pip3 install --upgrade --no-deps --force-reinstall nolearn
```
### Windows
```bat
# Python 2
python -m pip install --upgrade --no-deps --force-reinstall nolearn
# Python 3
python3 -m pip install --upgrade --no-deps --force-reinstall nolearn
```

**Comment by dnouri** [comment]
@nickmitchko To install the latest development version please refer to the [Installation Instructions](https://github.com/dnouri/nolearn#installation).
